lib/improver/uv_index.py

```diff
@@ -93,5 +93,7 @@ def calculate_uv_index(uv_upward, uv_downward, scale_factor=3.6):
         uv_index.data = (uv_upward.data +
                          uv_downward.data) * scale_factor
         uv_index.standard_name = "ultraviolet_index"
+        uv_index.long_name = "ultraviolet_index"
```

Comment:
Should the long_name just be set to None, as I don't think that we usually assign the same name to both the `standard_name` and the `long_name`?

Reply:
Done